owner-onboarding-server/src/main.rs

```diff
@@ -79,13 +90,129 @@ struct Settings {
 
     // Service Info
     service_info: crate::serviceinfo::ServiceInfoSettings,
+
+    // owner addresses path for report to rendezvous
+    owner_addresses_path: String,
 }
 
 fn load_private_key(path: &str) -> Result<PKey<Private>> {
     let contents = fs::read(path)?;
     Ok(PKey::private_key_from_der(&contents)?)
 }
 
+async fn report_to_rendezvous(udt: OwnerServiceUDT) -> Result<()> {
+    let mut ft = udt.ownership_voucher_store.query_data().await.unwrap();
+    ft.neq(
+        &fdo_store::MetadataKey::Local(OwnershipVoucherStoreMetadataKey::To2Performed),
+        &true,
+    );
+    ft.lt(
+        &fdo_store::MetadataKey::Local(OwnershipVoucherStoreMetadataKey::To0AcceptOwnerWaitSeconds),
+        chrono::Local::now().timestamp(),
+    );
+
+    let ov_iter = ft.query().await?;
+    if let Some(ovs) = ov_iter {
+        for ov in ovs {
+            let wait_seconds =
+                report_ov_to_rendezvous(&ov, &udt.owner_addresses, &udt.owner_key).await?;
```

Comment:
Perhaps make this a separate function, and log but not fail on error?
Otherwise, if one of the OV's fails, we also stop reporting the others.

Reply:
oh right, I'll just match here and log then